docs/File-Formats.md

```diff
@@ -331,6 +338,53 @@ AGRN<TAB>375790<TAB>2<TAB>0<TAB>...
 ...
 `````
 
+### Long format
+
+#### Meta file 
+The meta file of **wide format** is comprised of the following fields:
+
+1. **cancer_study_identifier**: same value as specified in [study meta file](#cancer-study)
+2. **genetic_alteration_type**: COPY_NUMBER_ALTERATION
+3. **datatype**: `DISCRETE_LONG` (versus `DISCRETE`)
```

Comment:
```suggestion
3. **datatype**: `DISCRETE_LONG`
```

Reply:
Fixed in [7e7ca7](https://github.com/cBioPortal/cbioportal/compare/646069c114345e4c1554f618c23396094122f4a0..7e7ca798ae5ac8bb041df89cfe1883252da1e964)